tests/lib/featcomposer.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+
+import argparse
+from collections import namedtuple
+import json
+import os
+import shutil
+from typing import Any
+import features
```

Comment:
Format nitpick, can you add a newline after all the standard library imports, to separate them from our own modules, please.